components/brave_shields/common/features.cc

```diff
@@ -32,6 +32,9 @@ const base::Feature kBraveAdblockCookieListDefault{
 const base::Feature kBraveAdblockCosmeticFiltering{
     "BraveAdblockCosmeticFiltering",
     base::FEATURE_ENABLED_BY_DEFAULT};
+const base::Feature kBraveAdblockCosmeticFilteringChildFrames{
+    "BraveAdblockCosmeticFilteringChildFrames",
+    base::FEATURE_ENABLED_BY_DEFAULT};
```

Comment:
until the perf issue is sorted out, would be better to ship disabled by default

cc @bbondy